src/modules/gateway/dto/initialize-workspace.dto.ts

```diff
@@ -9,6 +9,13 @@ export class InitializeWorkspaceDto {
   @IsNotEmpty()
   ownerId: string;
 
+  @ApiProperty({
+    example: 'Address from billing',
+    description: 'Address of the workspace',
+  })
+  @IsOptional()
+  address?: string;
```

Comment:
In PR #305 I ended up changing the db schema separating the address into its own table since according to issue PB-1556 it seemed the address was more complex than a simple string. That PR hasn't been reviewed yet but this conflicts with those changes.

Reply:
@jzunigax2 We have not implemented the billing module yet, specially taking into account this is an MVP.